src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -35,6 +35,7 @@ namespace Bit.Api.AdminConsole.Controllers;
 
 [Route("organizations/{orgId}/users")]
 [Authorize("Application")]
+[Authorize<MemberOrProviderRequirement>]
```

Comment:
We can add this one to the whole controller as an extra layer of protection - ensuring that you need to have some relationship to the org (as a member or provider) to access any endpoint within it. Then individual endpoints can have more specific requirements.